docs/install/ubuntu_setup.md

```diff
@@ -1,67 +1,103 @@
 # Installing MXNet on Ubuntu
 
-**NOTE:** For MXNet with Python installation, please refer to the [new install guide](http://mxnet.io/install/index.html).
+The following installation instructions are for installing MXNet on computers running **Ubuntu 16.04**. Support for later versions of Ubuntu is [not yet available](#contributions).
+<hr>
 
-MXNet currently supports Python, R, Julia, Scala, and Perl. For users of R on Ubuntu operating systems, MXNet provides a set of Git Bash scripts that installs all of the required MXNet dependencies and the MXNet library.
+## Contents
 
-The simple installation scripts set up MXNet for Python and R on computers running Ubuntu 12 or later. The scripts install MXNet in your home folder ```~/mxnet```.
+* [CUDA Dependencies](#cuda-dependencies)
+* [Quick Installation](#quick-installation)
+    * [Python](#install-mxnet-for-python)
+    * [pip Packages](#pip-package-availability)
+* [Standard Installation](#standard-installation)
+* [Installing Language Packages](#installing-language-packages-for-mxnet)
+    * [R](#install-the-mxnet-package-for-r)
+    * [Julia](#install-the-mxnet-package-for-julia)
+    * [Scala](#install-the-mxnet-package-for-scala)
+    * [Perl](#install-the-mxnet-package-for-perl)
+  * [Contributions](#contributions)
+  * [Next Steps](#next-steps)
 
-## Prepare environment for GPU Installation
+<hr>
 
-If you plan to build with GPU, you need to set up the environment for CUDA and CUDNN.
+## CUDA Dependencies
 
-First, download and install [CUDA 8 toolkit](https://developer.nvidia.com/cuda-toolkit).
+If you plan to build with GPU, you need to set up the environment for CUDA and cuDNN.
 
-Then download [cudnn 6](https://developer.nvidia.com/cudnn).
+First, download and install [CUDA toolkit](https://developer.nvidia.com/cuda-toolkit). CUDA 9.2 is recommended.
 
-Unzip the file and change to the cudnn root directory. Move the header and libraries to your local CUDA Toolkit folder:
+Then download [cuDNN 7.1.4](https://developer.nvidia.com/cudnn).
```

Comment:
Do we want to specify a minor cuDNN versions? We're always using the latest one in CI, so we shouldn't give the users the impression that only a certain one is supported

Reply:
I debated that, but I think if we're recommending an install path, then it should be the one we're testing all of the time. However, we should supply a table of the supported versions per release. I don't know what minor or major versions work for 1.1.0, 1.0.0, and so on....